compiler.config.json

```diff
@@ -54,7 +54,8 @@
                     "type": "files",
                     "files": [
                         "Freetext.txt",
-                        "Airpsace_Bases.txt"
+                        "Airspace_Bases.txt",
```

Comment:
Remove the airspace bases for now, while we sort out the compiler to be fine with them